cinema/serializers.py

```diff
@@ -77,4 +87,52 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
 
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall")
+        fields = ("id", "show_time", "movie", "cinema_hall", "taken_places")
```

Comment:
You added field taken_places, but model doesn't have this fields. You need to got it from tickets